server/api/login.js

```diff
@@ -83,10 +83,17 @@ internals.applyRoutes = function (server, next) {
                     });
                 }
             }, {
-                assign: 'session',
+                assign: 'existingSession',
```

Comment:
During a login request I don't think we should be looking up the existing session. As an API consumer, when I create a new login request, I would expect a new session to be created.